kernel/aster-nix/src/syscall/arch/x86.rs

```diff
@@ -180,9 +180,9 @@ impl_syscall_nums_and_dispatch_fn! {
     SYS_SOCKETPAIR = 53        => sys_socketpair(args[..4]);
     SYS_SETSOCKOPT = 54        => sys_setsockopt(args[..5]);
     SYS_GETSOCKOPT = 55        => sys_getsockopt(args[..5]);
-    SYS_CLONE = 56             => sys_clone(args[..5], &context);
-    SYS_FORK = 57              => sys_fork(args[..0], &context);
-    SYS_EXECVE = 59            => sys_execve(args[..3], &mut context);
+    SYS_CLONE = 56             => sys_clone(args[..5], current, &context);
+    SYS_FORK = 57              => sys_fork(args[..0], current, &context);
+    SYS_EXECVE = 59            => sys_execve(args[..3], current, &mut context);
     SYS_EXIT = 60              => sys_exit(args[..1]);
```

Comment:
Agree with @StevenJiang1110 's opinion that we should probably pass `ctx: Context` (see the other comment of mine about the name) to every system call function. I can hardly think of a system call that won't make use of `Context` if it is passed. For example, the signature of `sys_exit` is not modified. But in fact it benefits from having `ctx` so that `current_thread!` can be avoided.

```rust
pub fn sys_exit(exit_code: i32) -> Result<SyscallReturn> {
    debug!("exid code = {}", exit_code);

    let current_thread = current_thread!();

    // .. 
}
```